docker-compose.example.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.7"
+
+# NOTE: We don't use Docker in production at Retrospring, so this setup is
+#       provided as-is and might be broken or could be improved.
+#
+#       If you have experience in production docker setups, feel free to
+#       send in some patches!
+
+services:
```

Comment:
to be complete this needs to have a `sidekiq` service too

Reply:
What do we run in that service?